### PR TITLE
Amp 73976 ampli supported platforms on ampli overview

### PR DESCRIPTION
**Pull request by justin-fiedler**
## Description

* https://pr-837.d19s7xzcva2mw3.amplifyapp.com/data/sdks/ampli-overview/#supported-platforms
* Added list of supported platforms on Ampli overview
* Improved comments in Ampli code examples

image

### CODE REVIEW

File: includes/ampli/ampli-supported-platforms.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Ampli supports the following platforms - **Browser**, **Android**, **iOS**, **React Native**, **Node**, **Go**, **Python**, and **Java**.
+The other Amplitude SDKs for _Flutter_, _Unity_, and _Unreal_ _do not_ have Ampli at this time.
+
+??? note "Ampli Supported Platforms and SDKs"
+    | Platform     | Ampli Support | <div class="big-column">Supported Amplitude SDK(s)</div>                                                                                                                                                                                                                                                                                                                                                                                                  |
```
Comment (justin-fiedler) [inline includes/ampli/ampli-supported-platforms.md:5]: Not sure how to make the table full width. Any suggestions?

Comment (Mercy811) [reply includes/ampli/ampli-supported-platforms.md:5]: Can only make it full width with truncated lines between rows by simply adding to at the end of `docs/stylesheets/extra.css`

```.md-typeset__table {
  min-width: 100%;
}
```
image
Not sure if this is better how we are now though

**Review by Mercy811 — APPROVED** [review]
Thanks Justin for adding this table. Do we need ampli migration doc?

File: includes/ampli/ampli-supported-platforms.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    |--------------|---------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+    | Browser      | Yes           | :octicons-package-16: [`@amplitude/analytics-browser`](../typescript-browser/)<br/>:material-file-document: [Ampli documentation](../typescript-browser/ampli/)<br/><br/>:material-hammer-screwdriver:{.red}:octicons-package-16: [`com.amplitude:android-sdk`](../javascript/)<br/>:material-hammer-screwdriver:{.red}:material-file-document:[Ampli documentation](../javascript/ampli/)                  |
```
Comment (yuhao900914) [inline includes/ampli/ampli-supported-platforms.md:7]: `com.amplitude:android-sdk` should be the legacy javascript SDK?

File: includes/ampli/ampli-supported-platforms.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    | Android      | Yes           | :octicons-package-16: [`com.amplitude:analytics-android`](../android-kotlin/)<br/>:material-file-document: [Ampli documentation](../android-kotlin/ampli/)<br/><br/>:material-hammer-screwdriver:{.red}:octicons-package-16: [`com.amplitude:android-sdk`](../android/)<br/>:material-hammer-screwdriver:{.red}:material-file-document:[Ampli documentation](../android/ampli/)                             |
+    | iOS          | Yes           | :octicons-package-16: [`Amplitude`](./ios/)<br/>:material-file-document:[Ampli documentation](../ios/ampli/)                                                                                                                                                                                                                                                                                                        |
```
Comment (yuhao900914) [inline includes/ampli/ampli-supported-platforms.md:9]: the ios doc redirect to a wrong link.

File: includes/ampli/ampli-supported-platforms.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    | Python       | Yes           | :octicons-package-16: [`amplitude-analytics`](../python/)<br/>:material-file-document:[Ampli documentation](../python/ampli/)                                                                                                                                                                                                                                                                                        |
+    | Java         | Yes           | :octicons-package-16: [`com.amplitude.:java-sdk`](../java/)<br/>:material-file-document:[Ampli documentation](../java/ampli/)                                                                                                                                                                                                                                                                                        |
```
Comment (yuhao900914) [inline includes/ampli/ampli-supported-platforms.md:14]: `com.amplitude.:java-sdk` is this right?

**Pull request merged**